hardware_in_the_loop/software/requirements.txt

```diff
@@ -0,0 +1 @@
+python-can # current version is 3.3.4, idk if we are versioning in requirements
```

Comment:
thoughts on versioning?

Reply:
usually a good idea to ensure compatibility. I'd say just make it `python-can == 3.3.4` to be safe.

Reply:
Versioning everything now!